### PR TITLE
Updated support email ids, added additional images, and some small improvements.

### PR DESCRIPTION
**Pull request by srinivas-bitla**
### What does this PR do?

Updated support email ids, added additional images, and some small improvements.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: seagence/README.md
Comment (hestonhoffman) [inline seagence/README.md:15]:
```suggestion
After connecting your accounts, go to the "Assets" tab on the tile. Click **Recommended Monitors** > **Seagence Defect Detection Monitor**. This will redirect you to create the out-of-the-box monitor. Click **Create** at the bottom of the page to install the Seagence monitor.
```

File: seagence/README.md
Comment (hestonhoffman) [inline seagence/README.md:26]:
```suggestion
3. Delete the associated monitor by going to **Monitors** > **Manage Monitors**. Hover over **Seagence Defect Detection Monitor** and click **Delete**.
```

**Review by hestonhoffman — COMMENTED** [review]
Left you some style edits

**Comment by srinivas-bitla** [comment]
Added review recommendations.

**Review by emarsha94 — APPROVED** [review]

**Review by hestonhoffman — APPROVED** [review]

**Comment by srinivas-bitla** [comment]
Can I merge this PR? Not sure if I have write access.

**Comment by JoshPatel13** [comment]
@srinivas-bitla, will review first thing tomorrow and merge on behalf of ecosystems team

**Review by JoshPatel13 — APPROVED** [review]

**Pull request merged**
